x-pack/plugins/apm/server/routes/services/get_services/get_service_transaction_stats.ts

```diff
@@ -101,6 +102,11 @@ export async function getServiceTransactionStats({
                   size: maxNumServices,
                 },
                 aggs: {
+                  overflowCount: {
+                    max: {
+                      field: SERVICE_OVERFLOW_COUNT,
+                    },
+                  },
```

Comment:
I'm not 100% sure but I don't think we want to calculate this per service. Instead we want an overall overflow. Can you try moving this higher up?
Additionally, I don't get why we are doing a `max` agg. Shouldn't this be a `sum` of all the overflows counts combined? 

It doesn't look like we have any meaningful API tests for this. Can you please add an API test for this?

Reply:
If we have the `overflow_count` reported only for `_other` bucket, `sum` shouldn't be equal to `max` for this case?

Reply:
@sqren I had a discussion with @yngrdyn and we found that somehow we are unable to generate this data using syntrace at the moment. So in order to have better API Integration testing, we have created a Github Issue https://github.com/elastic/kibana/issues/149700 and we plan to address it during the test week.